.github/workflows/node.js.yml

```diff
@@ -1,7 +1,7 @@
 # This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
-name: Node.js CI
+name: Sclr.ac CI
```

Comment:
Node.js CI was a better name it defines what this action is, not what this project is 😅